src/mancala.cc

```diff
@@ -56,4 +61,37 @@ namespace mancala {
 				break;
 		}
 	}
-}  // namespace mancala
+
+	void Board::save() { 
+		fstream fout;
+		fout.open("saveState.csv", ios::out | ios::trunc);
+		for (int i = 0; i < sizeof(pockets); i++) { 
```

Comment:
Use `pockets.size()` instead of the `sizeof` operator. sizeof will give the size in bytes of the array, which for 32-bit ints will be 4 times the size of elements the array can hold.